contracts/src/v0.8/functions/dev/v1_0_0/FunctionsBilling.sol

```diff
@@ -218,26 +211,29 @@ abstract contract FunctionsBilling is Routable, IFunctionsBilling {
       callbackGasLimit: request.callbackGasLimit,
       estimatedTotalCostJuels: estimatedTotalCostJuels,
       timeoutTimestamp: uint32(block.timestamp + config.requestTimeoutSeconds),
-      requestId: requestId,
+      requestId: 0x0000000000000000000000000000000000000000000000000000000000000000,
```

Comment:
You don't have to set it, do you? Just remove and leave a comment that ID is left unset on purpose and set later.

Reply:
Turns out you do need to specify else Solidity gives a compilation error.